textattack/shared/attack.py

```diff
@@ -77,6 +77,9 @@ def __init__(
         self.search_method.get_transformations = self.get_transformations
         self.search_method.get_goal_results = self.goal_function.get_results
 
+        if isinstance(self.search_method, textattack.search_methods.GeneticAlgorithm):
+            self.search_method.filter_transformations = self.filter_transformations
```

Comment:
maybe we could just pass `filter_transformations` as an argument every time? I don't see a problem w that

Reply:
I guess we could. I was wondering if that would leave people confused as to whether they would need to `filter_transformations` after they get transformations from `get_transformations`